### PR TITLE
fix: SystemJS imports

### LINKED ISSUE #5922: [systemjs] Not transform `import.meta.fn()`
### Describe the bug

(I'm using Google Translate, sorry for any bad words)

This syntax is not translated when calling functions on `import.meta`.

### Input code

```typescript
import.meta.a
import.meta.fn()
import.meta.c
```

### Config

```json
{
  "jsc": {
    "parser": {
      "syntax": "typescript",
      "tsx": false
    },
    "target": "es2022",
    "loose": false,
    "minify": {
      "compress": false,
      "mangle": false
    }
  },
  "module": {
    "type": "systemjs"
  },
  "minify": false,
  "isModule": true
}
```

### Playground link

https://play.swc.rs/?version=1.3.2&code=H4sIAAAAAAAAA8vMLcgvKtHLTS1J1EvkykTipeVpaKIIJHMBAJOfj18tAAAA&config=H4sIAAAAAAAAA0WNSwrDMAxE76J1FsXL3KGHMK4SHOIPHgVqjO9euThkJ808PTU64GhtlG0BlzGhRrFfWklqZrjis9BCAo02e4K7LrbsLIowzMsYrc%2BUwBNYKPjotzpkLoVcGHgqG%2FfzJru6QvpcI2j%2Ff%2BpEhXA4QP0RzWOP96SlXNx%2FKcl2l70AAAA%3D

### Expected behavior

Transform to `_context.meta.fn()`.

### Actual behavior

_No response_

### Version

1.3.2

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by vasilev-alex**
**Description:**

Fixes dynamic SystemJS imports and also closes #5922

**Related issue (if exists):**

#5922

### CODE REVIEW

File: crates/swc_ecma_transforms_module/src/system_js.rs [hunk 1]
```diff
@@ -574,9 +589,12 @@ impl Fold for SystemJs {
                 _ => Expr::MetaProp(meta_prop_expr),
             },
             Expr::Await(await_expr) => {
+                let await_expr = await_expr.fold_children_with(self);
```
Comment (kdy1) [inline crates/swc_ecma_transforms_module/src/system_js.rs:592]: This seems like a double visit

File: crates/swc_ecma_transforms_module/src/system_js.rs [hunk 2]
```diff
@@ -610,6 +628,8 @@ impl Fold for SystemJs {
     }
 
     fn fold_prop(&mut self, prop: Prop) -> Prop {
+        let prop = prop.fold_children_with(self);
```
Comment (kdy1) [inline crates/swc_ecma_transforms_module/src/system_js.rs:631]: Same here

**Review by kdy1 — CHANGES_REQUESTED** [review]

File: crates/swc_ecma_transforms_module/src/system_js.rs (hunk 1 above)
Comment (vasilev-alex) [reply crates/swc_ecma_transforms_module/src/system_js.rs]: Yeah, you're right. But only here, not in `fold_prop`

**Review by kdy1 — APPROVED** [review]
Thank you!

---

swc-bump:
 - swc_ecma_transforms_module

**Pull request merged**